.github/workflows/boards.yml

```diff
@@ -1,35 +1,86 @@
 name: Automate Boards
 on:
   issues:
-    types: [opened, labeled]
+    types: [opened, labeled, milestoned, demilestoned]
+  issue_comment:
+    types: [created]
 jobs:
-  # move to triaging board if new
+  # move to Triaging board if new
   triaging:
     runs-on: ubuntu-latest
     steps:
-      - uses: alex-page/github-project-automation-plus@v0.8.1
+      - uses: alex-page/github-project-automation-plus@v0
```

Comment:
Let's... keep that pin and enable Dependabot or Renovate for GitHub actions, so we get action updates as PRs. E.g. https://docs.renovatebot.com/modules/manager/github-actions/

Reply:
ah hadn't thought of that, agreed